### PR TITLE
Updates feb 3

### PR DESCRIPTION
**Pull request by saschaben**

### CODE REVIEW

File: _posts/2017-02-03-updates.markdown [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title:  Terms of Reporting
+date:   2017-02-03 08:00:00
+categories: updates
+---
+Today we've updated to __v3.26.0__, which implements reports for all vocabulary terms in the MyReports tool. to find out just what reports are available, please visit [our reports grid](https://www.iliosproject.org/myreports.html).
```
Comment (Trott) [inline _posts/2017-02-03-updates.markdown]: to -> To

Comment (Trott) [inline _posts/2017-02-03-updates.markdown]: Remove `just`

File: _posts/2017-02-03-updates.markdown [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+We've also added a Docker version of the Ilios deployment ([contact us](mailto:support@iliosproject.org?subject=docker) for more info), and pushed out a bunch of great updates to the system, and would love to hear back from you about what you think! Write us at [support@iliosproject.org](mailto:support@iliosproject.org?subject=feedback)!
```
Comment (Trott) [inline _posts/2017-02-03-updates.markdown]: First sentence is a run-on. Put a period after `system` and make `We would love to hear back from you about what you think!` its own sentence. For that matter, shorten it to `We would love to hear what you think!` or even just `Tell us what you think!`

Comment (Trott) [inline _posts/2017-02-03-updates.markdown]: Remove `to the system`. First sentence can end after `updates`.

Comment (Trott) [inline _posts/2017-02-03-updates.markdown]: Remove `Write us` and have the last sentence just be something like `Tell us what you think at support@iliosproject.org!`

File: _posts/2017-02-03-updates.markdown (hunk 1 above)
Comment (saschaben) [inline _posts/2017-02-03-updates.markdown]: good catch. done/.

File: _posts/2017-02-03-updates.markdown (hunk 2 above)
Comment (saschaben) [inline _posts/2017-02-03-updates.markdown]: cleaned up. it'll do.

File: _posts/2017-02-03-updates.markdown [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title:  Terms of Reporting
+date:   2017-02-03 08:00:00
+categories: updates
+---
+Today we've updated to __v3.26.0__, which implements reports for all vocabulary terms in the MyReports tool. to find out what reports are available, please visit [our reports grid](https://www.iliosproject.org/myreports.html).
```
Comment (Trott) [inline _posts/2017-02-03-updates.markdown]: `to` -> `To`

Comment (saschaben) [inline _posts/2017-02-03-updates.markdown]: blurg. corrected

**Comment by Trott** [comment]
Text in the post doc LGTM. (Someone else will have to weigh in on the content and the other file.)

**Pull request merged**